src/control/server/ctl_storage_rpc.go

```diff
@@ -268,7 +268,7 @@ func (c *ControlService) StorageFormat(ctx context.Context, req *ctlpb.StorageFo
 	return resp, nil
 }
 
-// StorageNvmeRebind rebinds SSD from kernel and binds to user-space to allow DAOS to use it.
+// StorageNvmeRebind adds a newly added SSD to a DAOS engine's NVMe config to allow it to be used.
```

Comment:
Looks like this doc comment got switched with the one on the new command.

Reply:
whoops, thanks for the catch